evm/vm/evm.py

```diff
@@ -125,10 +106,10 @@ def get_vm_class_for_block_number(cls, block_number):
         """
         Return the vm class for the given block number.
         """
-        range = find_range(tuple(cls.vms_by_range.keys()), block_number)
-        base_vm_class = cls.vms_by_range[range]
-        vm_class = base_vm_class.configure(db=cls.db)
```

Comment:
Even in #12 I only see `get_vm_class_for_block_number()` used in regular instance methods, and I'm still not sure I understand in which situations you expect to have an EVM class being passed around -- ISTM that upon starting the client we'd always get one of the pre-configured EVM classes (e.g. MainnetEVM), create an instance of it (with a db) and then pass that around.  If that's the case we should be able to have just EVM.configure() as a @classmethod. Or am I missing something?